gravitee-apim-console-webui/src/management/api/general/details/api-general-info-duplicate-dialog/api-general-info-duplicate-dialog.component.ts

```diff
@@ -71,6 +72,26 @@ export class ApiGeneralInfoDuplicateDialogComponent implements OnDestroy {
         }, {}),
       ),
     });
+
+    if (dialogData.api.definitionVersion === 'V4') {
+      if (dialogData.api.listeners?.find((listener) => listener.type === 'TCP')) {
+        // FIXME: discuss during review
+        //  Should we reuse and mutualize what has been done in this PR ? https://github.com/gravitee-io/gravitee-api-management/pull/6298/files#diff-ca4491fa6ad94ef1e952b9c2e8a3bb7c354795e8c1a9783b93811cc9d23143e2R195-R204
+        //  or should we just rely on the backend response? as done for contextPath
+        //  My opinion is the better would be to have a sync validator in front side to prevent useless calls to the mAPI
+        this.duplicateApiForm.addControl('host', new UntypedFormControl('', [Validators.required], [this.apiV2Service.hostValidator({})]));
```

Comment:
I would have migrate the form in this pull request if you are ok. The last angular update as introduced UntypedFormControl to ease the migration but at imho we should think about adding typed controls ( like we did in the portal ).

I think we should discuss it in the team and introduce a new "development rule". When a developper work on an existing component with untyped form control the typing should be done. Or we will never do it.